### PR TITLE
docs(elevation API): add response explanation and confidence level definition

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: docs/MapColonies/DEM/Services/elevation/README.md [hunk 1]
```diff
@@ -22,13 +22,22 @@ Allows users to request elevation(height) information at a given location(s)/pos
...
+3. MIXED - according to most detailed (best resolution) materials.
 
-## Usage
+### Confidence level
+
+Calculated for each product and is represented by two values:
+* `resolutionMeter` - precision on the horizontal plane (may be interpolated), meaning that for every `x` meters there is a value in the mesh
+* `absoluteAccuracyLEP90` - each height measurement has a 90% chance to have a deviation of at most `x` meters from the given value (between `height` - `x` and `height` + `x`)
+
+It is extremely important to provide the user with this information for each point you query in order to allow him to make the best decision based on the data.
```
Comment (alebinson) [inline docs/MapColonies/DEM/Services/elevation/README.md:38]: It is extremely important to provide all information about product that was used for height extraction of queried location(point)

**Review by alebinson — DISMISSED** [review]
Basically approved, but see my comment (feel free to rephrase it)

File: docs/MapColonies/DEM/Services/elevation/README.md (hunk 1 above)
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/elevation/README.md:38]: Is this better?

File: docs/MapColonies/DEM/Services/elevation/README.md [hunk 2]
```diff
@@ -22,13 +22,22 @@ Allows users to request elevation(height) information at a given location(s)/pos
...
+3. MIXED - according to most detailed (best resolution) materials.
 
-## Usage
+### Confidence level
+
+Calculated for each product and is represented by two values:
+* `resolutionMeter` - precision on the horizontal plane (may be interpolated), meaning that for every `x` meters there is a value in the mesh
+* `absoluteAccuracyLEP90` - each height measurement has a 90% chance to have a deviation of at most `x` meters from the given value (between `height` - `x` and `height` + `x`)
+
+It is extremely important to provide all of the information about the product which the height was extracted from, for each point you query, in order to allow the user to make the best decision based on the data.
```
Comment (alebinson) [inline docs/MapColonies/DEM/Services/elevation/README.md:38]: GOOD, but think that part "in order to allow the user to make the best decision based on the data" is redundant

Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/elevation/README.md]: Removed, but added an `info` banner for the required values to be displayed.

File: docs/MapColonies/DEM/Services/elevation/README.md [hunk 3]
```diff
@@ -22,13 +22,26 @@ Allows users to request elevation(height) information at a given location(s)/pos
...
 2. DSM - surface materials.
-3. MIXED - according to most detailed(best resolution) materials.
+3. MIXED - according to most detailed (best resolution) materials.
 
-## Usage
+### Confidence level
+
+Calculated for each product and is represented by two values:
+* `resolutionMeter` - precision on the horizontal plane (may be interpolated), meaning that for every `x` meters there is a value in the mesh
+* `absoluteAccuracyLEP90` - each height measurement has a 90% chance to have a deviation of at most `x` meters from the given value (between `height` - `x` and `height` + `x`)
```
Comment (syncush) [inline docs/MapColonies/DEM/Services/elevation/README.md:36]:
```suggestion
* `absoluteAccuracyLEP90` - each height measurement has a 90% chance to have a deviation of at most `absoluteAccuracyLEP90` meters from the given value (between `height` - `absoluteAccuracyLEP90` and `height` + `absoluteAccuracyLEP90`)
```

Why use x ? it is confusing

File: docs/MapColonies/DEM/Services/elevation/README.md [hunk 4]
```diff
@@ -22,13 +22,26 @@ Allows users to request elevation(height) information at a given location(s)/pos
...
 1. DTM - terrain materials.
 2. DSM - surface materials.
-3. MIXED - according to most detailed(best resolution) materials.
+3. MIXED - according to most detailed (best resolution) materials.
 
-## Usage
+### Confidence level
+
+Calculated for each product and is represented by two values:
+* `resolutionMeter` - precision on the horizontal plane (may be interpolated), meaning that for every `x` meters there is a value in the mesh
```
Comment (syncush) [inline docs/MapColonies/DEM/Services/elevation/README.md:35]: same as absoluteAccuracyLEP90, why use x ?

**Review by syncush — CHANGES_REQUESTED** [review]
Where is the open api for elevation ?

File: docs/MapColonies/DEM/Services/elevation/README.md (hunk 3 above)
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/elevation/README.md]: Fixed.

File: docs/MapColonies/DEM/Services/elevation/README.md (hunk 4 above)
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/elevation/README.md]: Fixed.

**Pull request merged**